websites/R/Rumble/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "562755652921982986",
+		"name": "PrimeMinus"
+	},
+	"service": "Rumble",
+	"description": {
+		"en": "Rumble is an alternate online video sharing platform for people with something to say and something to share, who believe in authentic expression, and want to control the value of their own creations."
+	},
+	"url": "rumble.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/yGiywx1.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px